lib/chibi/sxml.scm

```diff
@@ -93,6 +93,9 @@
                   (display (html-tag->string tag (cdar rest)) out)
                   (for-each lp (cdr rest))
                   (display "</" out) (display tag out) (display ">" out))
+                 ((and (eq? 'safe-string tag)
```

Comment:
I think "raw" or "unescaped" would be a better term - this is explicitly an unsafe operation!

It's also probably better to avoid collision with valid tags, maybe `(@unescaped <str>)`.

Reply:
Updated and chose @raw as you said.